addons/api/addon/services/indexed-db.js

```diff
@@ -29,6 +29,8 @@ export const modelIndexes = {
     '&id, attributes.created_time, attributes.type, attributes.name, attributes.description, attributes.scope.scope_id, attributes.plugin.name',
   'auth-method':
     '&id, attributes.created_time, attributes.type, attributes.name, attributes.description, attributes.is_primary, attributes.scope.scope_id',
+  worker:
```

Comment:
We will need to update the version number if we add a new index as this would break for any existing user that already has an indexed db

Reply:
Thanks for the reminder. Version has been updated.